song.py

```diff
@@ -0,0 +1,172 @@
+
+# save strings as variables and then join them together
+# use f-strings to format so the main lyric stays the same but the ending changes
+# at the end of every chunk: I don't know why she swallowed the fly. Perhaps she'll die.
+
+################################################################
+
+# Here are my initial ideas:
+
+# part_verse_1 = "I know an old lady who swallowed a "
+#   full_verse_1 = part_verse_1 + "fly"
+#   return full_verse_1
+
+# def verse2():
+#   part_verse_2 = part_verse_1
+#   end_verse_2 = part_verse_2 + "spider"
+#   return(end_verse_2)
+# print(end_verse_1)
+
+################################################################
+
+# Here I am thinking about how to do the same lyrics but using a dictionary with key and value pairs
+
+# variations = {
+#         "verse_2": ["bird", "How absurd to swallow a bird!"], 
+#         "verse_3": ["cat", "Imagine that, to swallow a cat!"]
+#           }
+
+################################################################
+
+animals = ["fly", "spider", "bird", "cat", "dog", "goat", "cow"]
+refrain = "I know an old lady who swallowed a"
+line_7 = "I know an old lady who swallowed a horse. She's dead, of course!"
+line_6 = "She swallowed the cow to catch the goat."
+line_5 = "She swallowed the goat to catch the dog."
+line_4 = "She swallowed the dog to catch the cat."
+line_3 = "She swallowed the cat to catch the bird."
+line_2 = "She swallowed the bird to catch the spider that wriggled and jiggled and tickled inside her."
+line_1 = "She swallowed the spider to catch the fly."
+close = "I don't know why she swallowed the fly. Perhaps she'll die. "
+
+# This was my first working attempt with an f-string, getting the list of animals to repeat with the same refrain and close.
+# def verse(animal):
+#     for animal in animals:
+#         print(f"{refrain} {animal}. {close}")      
+# verse(refrain)
+
+# Now here is basic working code for the assignment that at least uses f-strings
+
+def verse_1(spider):
```

Comment:
This is effective problem-solving. Nice job identifying the patterns and using f strings and variables!